microcosm_sqlite/builders/csv.py

```diff
@@ -16,10 +16,12 @@ def __init__(
         graph,
         model_cls,
         bulk_mode=False,
+        debug_mode=False,
```

Comment:
I'd prefer to name this something more specific since we already have `debug` in the object graph metadata and it probably doesn't imply this specific behavior every time.

In addition, the behavior you want might just be `bulk_mode=False`?

Reply:
@jessemyers
1. How about  `commit_on_insert` instead of debug_mode?
2. I thought so re `bulk_mode` but it doesnt seem to be the case (Bulk seems to refer to a bulk of different models rather than individual rows)